voice-client-javascript/assets/admin/statuses.private.js

```diff
@@ -0,0 +1,256 @@
+const assets = Runtime.getAssets();
+const { getCurrentEnvironment } = require(assets["/admin/environment.js"].path);
+
+async function checkEnvironmentInitialization(context) {
+  const environment = await getCurrentEnvironment(context);
+  const status = {
+    title: `Environmental Checks`,
+    valid: false
+  };
+  if (!environment) {
+    status.description = `This application is **must be** deployed. 
+
+
+To deploy this function, use the following command:
+
+\`\`\`bash
+twilio serverless:deploy
+\`\`\`
+
+After it has been deployed, revisit this page in your deployed application.
+`;
+  } else if (!process.env.INITIALIZED) {
+    status.description = `The Twilio Client JavaScript Quickstart requires that you setup a few things on your account. 
+
+We've written some tools that will initialize the various parts to use this tool.
+
+To initialize your environment, click the button below.
+
+We'll explain what each of these parts are after we get started.`;
+    status.actions = [
+      {
+        title: `Initialize your application for your environment, ${environment.uniqueName}`,
+        name: "initialize"
+      }
+    ];
+  } else {
+    status.valid = true;
+    status.description = `Your application is initialized! View your [running application](/index.html)`;
```

Comment:
This doesn't work for nested templates. For example if someone created this under a namespace using `twilio serverless:new`. I'd change the path from `/index.html` to `../index.html` and this should do the trick